tutorials/introductory/customizing.py

```diff
@@ -61,7 +51,25 @@
 # good in a presentation, you can just add::
 #
 #    >>> import matplotlib.pyplot as plt
-#    >>> plt.style.use('presentation')
+#    >>> plt.style.use('../images/presentation.mplstyle')
+#
+# Alternatively, you can load your custom style sheet with a call to
```

Comment:
I would write it the other way round, something along the lines:

> Alternatively, you can place your ``<style-name>.mplstyle`` file into
``mpl_configdir/stylelib``, and then load it via ``style.use(<style-name>)``.

Reply:
I've changed the text to read as you've suggested here - I think your text is clearer.